### PR TITLE
Updates GCS, Spanner and BigQuery source plugin widget jsons to have …

### PR DESCRIPTION
**Pull request by ajainarayanan**
…browse widget associated with projectId

### CODE REVIEW

File: widgets/BigQueryTable-batchsource.json [hunk 1]
```diff
@@ -21,6 +21,13 @@
           "name": "project",
           "widget-attributes" : {
             "default": "auto-detect"
+          },
+          "plugin-function": {
```
Comment (albertshau) [inline widgets/BigQueryTable-batchsource.json:25]: does this get ignored by earlier versions of the UI?

**Review by albertshau — APPROVED** [review]
lgtm assuming it doesn't break earlier versions of the UI

File: widgets/BigQueryTable-batchsource.json (hunk 1 above)
Comment (ajainarayanan) [reply widgets/BigQueryTable-batchsource.json:25]: The plugin function as a concept got removed in the backend. I don't believe we changed the spec to reflect that. In plugins like [Database source](https://github.com/cdapio/hydrator-plugins/blob/develop/database-plugins/widgets/Database-batchsource.json#L47) it still exists.

**Comment by ajainarayanan** [comment]
@albertshau Good point. Right now it does throw an exception but this will not be visible to the user nor will it be blocking the user flow. It should be fine if the plugin is used in earlier versions of CDAP (6.0). Have filed a [JIRA-CDAP-15592](https://issues.cask.co/browse/CDAP-15592)  anyways to fix this in 6.0 and 6.1

**Pull request merged**